### PR TITLE
Update self-hosted.rst

### PR DESCRIPTION
**Pull request by Ole-Morten**
Added docker compose section, removed agent. We still need to add BYOK for compose.

### CODE REVIEW

File: hub/documentation/operations/files/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# IMAGE TAGS, USER_ID and HOST names is found in .env file
+services:
+  watchtower:
+    image: containrrr/watchtower
+    container_name: watchtower
+    restart: always
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /home/sesam/.docker/config.json:/config.json
+    command: sesam-node redis solr databrowser fluentbit traefik
```
Comment (heneri) [inline hub/documentation/operations/files/docker-compose.yaml:10]: remove redis, solr, databrowser

File: hub/documentation/operations/files/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  sesam-node:
+    image: sesam/sesam-node:${SESAM_NODE_IMAGE_TAG}
+    container_name: sesam-node
+    restart: always
+    networks:
+      - sesam
+      - microservices
+    ports:
+      - "9042:9042"
+    volumes:
```
Comment (heneri) [inline hub/documentation/operations/files/docker-compose.yaml:50]: remove ports as we don't need to expose 9042 on hostlevel(0.0.0.0)

File: hub/documentation/operations/files/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    container_name: traefik
+    restart: always
+    command:
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--certificatesresolvers.myleresolver.acme.httpchallenge=true"
+      - "--certificatesresolvers.myleresolver.acme.httpchallenge.entrypoint=web"
+      - "--certificatesresolvers.myleresolver.acme.email=sesamadmin@sesam.io"
```
Comment (heneri) [inline hub/documentation/operations/files/docker-compose.yaml:27]: not sure if we should add the sesam email in the example. Maybe dnsadmin@yourdomain.com or something. We don't want letsencrypt/cert notifications for the customers.

**Review by Ole-Morten — COMMENTED** [review]
I think its good enough to publish and then update based on questions from bane nor as they pop up.

**Review by grove — APPROVED** [review]

**Pull request merged**